zulipterminal/model.py

```diff
@@ -502,6 +507,23 @@ def get_messages(self, *,
         display_error_if_present(response, self.controller)
         return response['msg']
 
+    @staticmethod
+    def modernize_message_response(response: List[Message]) -> List[Message]:
+        """
+        Cleans recieved messages into the modern format
```

Comment:
This docstring could be more helpful - why is it useful/helpful?

Also "receive".